jina/jaml/parsers/default/v1.py

```diff
@@ -5,6 +5,7 @@
 
 
 class V1Parser(VersionedYAMLParser):
+    """V1 default parser, used as the default parser for objects except driver, executor and flow."""
```

Comment:
Are we sure is not used for Flow if version is set to 1?

Reply:
To my understanding, this is the V1 default parser, and we have another v1 parser for flow. 
![image](https://user-images.githubusercontent.com/44033547/108443768-5c22af00-7227-11eb-8bd5-1b059fd7c16a.png)

Reply:
I updated the expression in the new commit.